deploy/aws/stacks-eks/example/terraform.tfvars

```
@@ -33,17 +33,18 @@ map_users = [
     username = "william.ayerst@amido.com"
     groups   = ["system:masters"]
   },
-  {
-    userarn  = "arn:aws:iam::316154162729:user/darren.smallwood@amido.com"
-    username = "darren.smallwood@amido.com"
-    groups   = ["system:masters"]
-  },
   {
     userarn  = "arn:aws:iam::316154162729:user/terraform"
     username = "terraform"
     groups   = ["system:masters"]
   }
 ]
+map_roles = [{
+  groups   = ["system:masters"]
+  rolearn  = "arn:aws:iam::316154162729:role/eks-admin-role"
+  username = "eks-admin-role"
+}]
```

This is just me being pinickity. But the rest go in order: userarn. username, groups. Could we re-order this to match? 
```suggestion
map_roles = [{
  rolearn  = "arn:aws:iam::316154162729:role/eks-admin-role"
  username = "eks-admin-role"
  groups   = ["system:masters"]
}]
```

---

will fix in the next PR as I need to redo for all the environments (that is dev and prod at once)